utils.js

```diff
@@ -0,0 +1,15 @@
+import axios from 'axios'
+
+export const getReviews = (id) => {
+    if(id){
+        return axios
+        .get(`https://gameproject-nd4p.onrender.com/api/reviews/${id}`)
+        .then((response) =>{ 
+          return response.data.review[0]}); 
+    }else
+      return axios
+      .get(`https://gameproject-nd4p.onrender.com/api/reviews/`)
+      .then((response) =>{ 
+        return response.data.reviews});
```

Comment:
Please separate this out into two separate functions. You should have one to get all the reviews and a separate one to get a single review by id.